src/iop/channelmixerrgb.c

```diff
@@ -3743,11 +3743,11 @@ void _auto_set_illuminant(dt_iop_module_t *self, dt_dev_pixelpipe_t *pipe)
   dt_Lab_2_LCH(Lab, Lch);
 
   // Write report in GUI
+  gchar *str = g_strdup_printf(_("L: \t%.1f %%\nh: \t%.1f °\nc: \t%.1f"), Lch[0], Lch[2] * 360.f, Lch[1]);
   ++darktable.gui->reset;
```

Comment:
No need to bracket pure gtk calls in `++/--darktable.gui->reset;` 
Can be cleaned up.

Reply:
It's in a quite a few places actually. As it is not related to leaks, I'd prefer that cleanup happens in its own, separate PR.

Reply:
I should have been more specific; pure gtk calls _without side effects_. So for example `gtk_toggle_button_set_active` will usually trigger a signal which may need to be deactivated via darktable.gui->reset.

I believe this particular instance was a leftover from a refactoring where previously the gui->reset bracketed a bunch of calls-with-sideeffects but that is no longer the case. No need to clean up now, if you want to keep the PR to its essentials.